backend/src/main/java/pl/krkteam/battleships/wait/opponent/dto/WaiterResponseDTO.java

```diff
@@ -2,4 +2,8 @@
 
 public abstract class WaiterResponseDTO {
     String response;
```

Comment:
private

Reply:
the same as above - it's in abstract class